bin/describe.ml

```diff
@@ -697,6 +890,16 @@ module What = struct
       , "builds a given FILE and prints the preprocessed output"
       , let+ s = filename in
         Pp s )
+    ; ( "external-lib-deps"
+      , []
+      , "Print out external libraries needed to build the project. It's an \
+         approximated set of libraries."
+      , return External_lib_deps )
+    ; ( "missing-external-lib-deps"
```

Comment:
does `dune-opam-lint` require this second one? we said we'd start with the bare minimum to support that command for now.

Reply:
No. I'm going to retract that.

Reply:
Yeah, even with only external libs we can find missing one without `dune`.